assets/Config/languages.json

```diff
@@ -108,6 +108,22 @@
                             "عملیات"
                         ]
                     }
+                },
+                "competitors": {
+                    "header": {
+                        "button": "افزودن رقیب",
+                        "competitor_counter": "رقیب شماره [number]"
+                    },
+                    "table": {
+                        "columns_1": [
```

Comment:
Due to the utilization of static templates in this section, please use semantic 'keys' to indicate the translation for a specific phrase

Reply:
Yes, I completely agree. Since the text in this section is set as a column, I hadn't a better idea for naming it. If you have a better suggestion for naming this section, please let me know in the feedback, and I will make the necessary corrections.